Description: Disable doclint and change the source level to fix the build failure with OpenJDK 17
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: https://github.com/junit-team/junit4/issues/1696
Bug-Debian: https://bugs.debian.org/981830
--- a/pom.xml
+++ b/pom.xml
@@ -530,14 +530,14 @@
                         <configuration>
                             <compilerArgs>
                                 <arg>-Xlint:unchecked</arg>
-                                <arg>-Xdoclint:accessibility,reference,syntax</arg>
+                                <arg>-Xdoclint:none</arg>
                             </compilerArgs>
                         </configuration>
                     </plugin>
                     <plugin>
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
-                            <additionalparam>-Xdoclint:accessibility -Xdoclint:reference</additionalparam>
+                            <additionalparam>-Xdoclint:none</additionalparam>
                         </configuration>
                     </plugin>
                 </plugins>
@@ -547,7 +547,7 @@
                     <plugin>
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
-                            <additionalparam>-Xdoclint:accessibility -Xdoclint:reference</additionalparam>
+                            <additionalparam>-Xdoclint:none</additionalparam>
                         </configuration>
                     </plugin>
                 </plugins>
@@ -567,7 +567,7 @@
                     <plugin>
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
-                            <source>1.6</source>
+                            <source>8</source>
                         </configuration>
                     </plugin>
                 </plugins>
@@ -577,7 +577,7 @@
                     <plugin>
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
-                            <source>1.6</source>
+                            <source>8</source>
                         </configuration>
                     </plugin>
                 </plugins>
